ChatUI docker template docs (#892)
huggingface/hub-docs: docs/hub/spaces-sdks-docker-chatui.md

# Chat UI on Spaces

> **Hugging Chat** is the interface of Hugging Face enabling everyone to try open-source large language models using Hugging Face's infrastructure. This Docker template enables everyone to deploy their own Hugging Chat based on a model of their choice with a few clicks.

Reviewer: WDYT of these additions?
```suggestion
**Hugging Chat** is an open-source interface enabling everyone to try open-source large language models such as Falcon, StarCoder and BLOOM. Thanks to an official Docker template, you can deploy your own Hugging Chat based on a model of your choice with a few clicks using Hugging Face's infrastructure.
```

Reviewer 2: or better the other way around:

> chat-ui is an open source interface [...] It is the codebase that powers HuggingChat: (link to huggingchat)

Also please mention OpenAssistant (and LLaMA too)

## Deploy your own Chat UI

> To get started, simply head [here](https://huggingface.co/new-space?template=huggingchat/chat-ui-template). You can select the hardware to serve the model of your choice.

Reviewer 2: ```suggestion
To get started, simply head [here](https://huggingface.co/new-space?template=huggingchat/chat-ui-template). You can select the hardware of your choice (GPU) which will serve the model inference.
```

<a href="https://huggingface.co/new-space?template=huggingchat/chat-ui-template">
    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/huggingface.co_spaces_docker_chatui_landing.png" />
</a>

> You should provide a MongoDB endpoint where your chats will be written. Note that Hugging Face does not have access to your chats. Below this, you can select the Hugging Face Hub ID of the model you wish to serve. To provide a database You can also change the generation hyperparameters in the dictionary below. 

Reviewer: ```suggestion
You can **optionally** provide a MongoDB endpoint where your chats will be written. Note that Hugging Face does not have access to your chats. Below this, you can customize your Hugging Chat by selecting the repository ID of the model you wish to serve. You can also change the generation parameters and customize UI elements.
```

Reviewer 3: Might be worth mentioning that with the new persistent storage for spaces, you can just leave the URL blank and still have a persistent DB for your chats!

Reviewer 2: yes, 100%

(feel free to push your changes to the doc directly if @merveenoyan agrees)

Author: @nsarrazin you can go ahead :)

Author: I addressed this.

<a href="Parameters">
    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/huggingface.co_spaces_docker_chatui_params.png" />
</a>

> Once the creation is complete, you will see `Building` on your Space. After that, you can try your own HuggingChat.

Reviewer: ```suggestion
Once the creation is complete, you will see `Building` on your Space. Once built, you can try your own HuggingChat!
```

<a href="Hugging Chat Landing UI">

>     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/huggingface.co_spaces_docker_chatui_ui.png" />

Reviewer 2: suggestion: use a differnt/more unique name than ChatUI so that people can see they can customize easily (chat-ui being the name of the repo)

Author: You mean in the filename inside the dataset?

Author: ah wait you mean the title?

Reviewer 2: yep the configurable title

Author: ah ok

Author: I'm swapping the images directly in the dataset (soon as my customized Space builds I'll also swap the UI screenshot)

</a>

Start chatting!

<a href="Hugging Chat">
    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/huggingface.co_spaces_docker_chatui_chat.png" />
</a>

## More on HF Spaces

> - [HF Docker spaces](https://huggingface.co/docs/hub/spaces-sdks-docker)

Reviewer: ```suggestion
- [HF Docker Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker)
```

Reviewer 2: TODO: also link to the GitHub repo of the open source app https://github.com/huggingface/chat-ui

# ChatUI on Spaces

> **Hugging Chat** is an open-source interface enabling everyone to try open-source large language models such as Falcon, StarCoder, and BLOOM. Thanks to an official Docker template, you can deploy your own Hugging Chat based on a model of your choice with a few clicks using Hugging Face's infrastructure.

Reviewer 2: ```suggestion
**Hugging Chat** is an open-source interface enabling everyone to try open-source large language models such as Falcon, StarCoder, and BLOOM. Thanks to an official Docker template called ChatUI, you can deploy your own Hugging Chat based on a model of your choice with a few clicks using Hugging Face's infrastructure.
```
